sh.exe.stackdump

```diff
@@ -0,0 +1,11 @@
+Stack trace:
+Frame        Function    Args
+00000010000  0018006137E (0018025BEB0, 0018024E199, 00000010000, 000FFFFBA00)
+00000010000  00180049229 (00100002000, 00000000002, 00000000002, 00180339770)
+00000010000  00180049262 (00000000002, 00180339A80, 00000010000, 00000000008)
+00000010000  0018005A9EC (00000000000, 0018024E48F, 0018004A6FE, 635C655C785C655C)
+000FFFFCCD0  0018005AA90 (745C685C275C205C, 3A5C735C705C745C, 695C675C2F5C2F5C, 625C755C685C745C)
+000FFFFCCD0  001800499EC (00000000000, 00000000000, 00000000000, 00000000000)
+00000000000  00180048343 (00000000000, 00000000000, 00000000000, 00000000000)
```

Comment:
I suppose this is not a file which we need and should merge.

Reply:
Yes, this is an accident. I deleted the file.